src/deepsparse/transformers/pipelines/text_generation.py

```diff
@@ -412,30 +408,96 @@ def output_schema(self) -> Type[BaseModel]:
         """
         return TextGenerationOutput
 
+    def _process_generation_config(
```

Comment:
as this script grows larger and larger, one thing that can help with readability is moving the hidden methods to the bottom of the script

Reply:
sounds good